airbyte-webapp/src/views/Connector/ConnectorForm/components/Sections/auth/AuthButton.tsx

```diff
@@ -14,19 +15,21 @@ import GoogleAuthButton from "./GoogleAuthButton";
 import { useFormikOauthAdapter } from "./useOauthFlowAdapter";
 
 function isGoogleConnector(connectorDefinitionId: string): boolean {
-  return [
-    "253487c0-2246-43ba-a21f-5116b20a2c50", // google ads
-    "eff3616a-f9c3-11eb-9a03-0242ac130003", // google analytics
-    "d19ae824-e289-4b14-995a-0632eb46d246", // google directory
-    "eb4c9e00-db83-4d63-a386-39cfa91012a8", // google search console
-    "71607ba1-c0ac-4799-8049-7f4b90dd50f7", // google sheets source
-    "a4cbd2d1-8dbe-4818-b8bc-b90ad782d12a", // google sheets destination
-    "ed9dfefa-1bbc-419d-8c5e-4d78f0ef6734", // google workspace admin reports
-    "afa734e4-3571-11ec-991a-1e0031268139", // YouTube analytics
-    // TODO: revert me
-    "78752073-6d96-447d-8a93-2b6953f3c787", // Youtube analytics Business
-    //
-  ].includes(connectorDefinitionId);
+  return (
+    [
+      ConnectorIds.Sources.GoogleAds,
+      ConnectorIds.Sources.GoogleAnalyticsUniversalAnalytics,
+      ConnectorIds.Sources.GoogleDirectory,
+      ConnectorIds.Sources.GoogleSearchConsole,
+      ConnectorIds.Sources.GoogleSheets,
+      ConnectorIds.Sources.GoogleWorkspaceAdminReports,
+      ConnectorIds.Sources.YouTubeAnalytics,
+      ConnectorIds.Destinations.GoogleSheets,
+      // TODO: revert me
+      ConnectorIds.Sources.YouTubeAnalyticsBusiness, // Youtube analytics Business
+      //
```

Comment:
```suggestion
      ConnectorIds.Sources.YouTubeAnalyticsBusiness,
```
any reason to keep those comments?

Reply:
I think this is still meant to be removed, and currently only enabled temporarily, for Google to validate the connector. My understanding was, that the plan is to remove that connector altogether again.

Reply:
I think it makes sense to keep the `// TODO: revert me` comment, but I just meant we can remove the `// Youtube analytics Business` comment, because we now have the name of it in the ConnectorIds value reference, and we can also remove that empty `//` on the line below

Reply:
Ah thank you. Misunderstood that. Yeah they are useless now. Will remove.